bin/tape

```diff
@@ -1,96 +1,5 @@
 #!/usr/bin/env ruby
 
-$LOAD_PATH << File.join(__dir__, '..', 'lib')
+require 'taperole'
 
```

Comment:
:) That's the "right" :tm: way to do things. aka: I copied it from some other gems.